packages/core/src/components/accordion/accordion.scss

```diff
@@ -0,0 +1,16 @@
+@import "~@dso-toolkit/sources/src/styling/variables";
+@import "~@dso-toolkit/sources/src/styling/mixins";
+
+@import "~@dso-toolkit/sources/src/utilities";
+
+@import "~@dso-toolkit/sources/src/components/accordion/accordion";
+
+@import "../../icon/di";
```

Comment:
Hier hadden we het vanmiddag al even over, `@mixin di()` is verhuisd naar `packages/css` en moet ook in die package worden aangeroepen.